AssetManagement.Models/StateRecord.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+
+namespace AssetManagement.Models
+{
+    public enum AssetState { Missing, Recovered }
```

Comment:
Change name of Recovered

```suggestion
    public enum AssetState { Missing, Online }
```